### ISSUE TITLE
indentation rules shouldn't apply to fonts

### ISSUE THREAD

**Issue by mmase** [sasstools/sass-lint#900]
The following throws warnings on indentation when it likely shouldn't:

```scss
@font-face {
  font-family: 'Icons';
  src: url('/assets/fonts/Icons/Icons.eot');
  src: url('/assets/fonts/Icons/Icons.eot?#iefix') format('embedded-opentype'),
       url('/assets/fonts/Icons/Icons.woff') format('woff'),
       url('/assets/fonts/Icons/Icons.ttf') format('truetype'),
       url('/assets/fonts/Icons/Icons.svg#Website') format('svg');
  font-weight: normal;
  font-style: normal;
}
```

**Comment by DanPurdy** [comment]
this is a duplicate of #591
